pudb/debugger.py

```diff
@@ -186,8 +186,6 @@ class Debugger(bdb.Bdb):
     def __init__(self, stdin=None, stdout=None, term_size=None, steal_output=False,
                  _continue_at_start=False, **kwargs):
 
-        if Debugger._current_debugger:
-            raise ValueError("a Debugger instance already exists")
```

Comment:
The problem with this is that various parts of the code assume that only one of these objects ever exists, e.g. here:

https://github.com/inducer/pudb/blob/b9c19487fa2d85c3da2eabcfa79f6732df4fe79d/pudb/__init__.py#L74-L90

Reply:
I see... can you think of way to fix the issue without messing with those parts of the code? Maybe we can destroy the pudb instance when `c` is pressed?